View/DataContexts/SongListDataContext.cs

```diff
@@ -29,10 +30,16 @@ public static SongListDataContext Instance
             }
         }
 
+
         public ScreenNames ScreenName { get; set; }
 
+        public List<string> SongListSearchTerms { get; set; } = new List<string>(){""};
```

Comment:
Waar wordt dit geupdate?

Reply:
En waarom zou je de searchterms niet gewoon meegeven aan de Constructor van `UpdateSongInfoList`?

Reply:
dit word geupdate in mainwindow op basis van welk scherm je zit, als ik hem aan de constructor meegeef dan word ie maar een keer geupdate.

Reply:
okaay